_howdoi/choose-my-spark-distribution.adoc

```diff
@@ -0,0 +1,68 @@
+= How do I choose my own Apache Spark distribution?
+:page-layout: howdoi
+:page-menu_entry: How do I?
+
+When you install radanalytics.io, your project is set up to use a pre-determined distribution of Apache Spark. You can choose your own Apache Spark distribution with a few additional steps:
+
+.Prerequisites
+
+. Follow the instructions under link:/howdoi/install-radanalyticsio[How do I install radanalytics.io?]
+
+. Choose an Apache Spark binary tarball like link:https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.7.tgz[this one]. You can download it or just use the url. Any binary distribution should work as long as it has this basic structure:
```

Comment:
same here, we should start with the prereq then lead into the discussion. so the prereq is "An Apache Spark tarball for the release you would like to deploy. Choose an Apache Spark tarball like...."

Reply:
okay, gotcha, thanks